.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
I don't think we need this. Let's stick with consistent configuration as the other connectrpc Go library repos.